RetailCoder.VBE/UI/Command/ExportAllCommand.cs

```diff
@@ -0,0 +1,77 @@
+using System.IO;
+using System.Windows.Forms;
+using NLog;
+using Rubberduck.VBEditor.SafeComWrappers.Abstract;
+using Rubberduck.Navigation.CodeExplorer;
+using Rubberduck.UI.CodeExplorer.Commands;
+
+namespace Rubberduck.UI.Command
+{
+    [CodeExplorerCommand]
+    public class ExportAllCommand : CommandBase 
```

Comment:
@Vogel612 indeed... but #3070 implies a complete overhaul of how we solve the hotkey problem. I think for now it would be simpler to just map a hotkey under the current system.

@shadowofsilicon I've added a [comment on the issue](https://github.com/rubberduck-vba/Rubberduck/issues/3070#issuecomment-322774712) that explains what's needed to associate a hotkey with a command. It's a bit clunky indeed, but it's not hard at all.